Keas.Mvc/ClientApp/components/People/PersonDetails.tsx

```diff
@@ -2,34 +2,32 @@ import * as PropTypes from 'prop-types';
 import * as React from "react";
 import {
     Button,
-    Modal,
-    ModalBody,
-    ModalFooter,
-    ModalHeader,
 } from "reactstrap";
-import { IPerson } from "../../Types";
+import { IPerson, IPersonInfo } from "../../Types";
 import AccessContainer from "../Access/AccessContainer";
-import BioContainer from "./BioContainer";
 import EquipmentContainer from "../Equipment/EquipmentContainer";
 import HistoryContainer from "../History/HistoryContainer";
 import KeySerialContainer from "../Keys/KeySerialContainer";
 import WorkstationContainer from "../Workstations/WorkstationContainer";
+import BioContainer from "./BioContainer";
+import DeletePerson from './DeletePerson';
 import EditPerson from "./EditPerson";
 
 interface IProps {
     goBack: () => void;
-    selectedPerson: IPerson;
+    selectedPersonInfo: IPersonInfo;
     tags: string[];
     inUseUpdated: (type: string, spaceId: number, personId: number, count: number) => void;
     edited?: (type: string, spaceId: number, personId: number) => void;
     onEdit: (person: IPerson) => void;
+    onDelete: (person: IPerson) => void;
 }
 
 
 export default class PersonDetails extends React.Component<IProps, {}> {
 
     public render() {
-        if (this.props.selectedPerson == null)
+        if (this.props.selectedPersonInfo == null || this.props.selectedPersonInfo.person == null)
```

Comment:
@laholstege I know it was like that before but try to dump the double equals if you can.  Either use `===` or cleaner yet just use `!` because `if (!this.props.selectedPersonInfo)` should get you the same result